src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -396,7 +409,14 @@ private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         volumes.add(new VolumeBuilder().withName(WORKSPACE_VOLUME_NAME).withNewEmptyDir("").build());
         Map<String, Container> containers = new HashMap<>();
 
+        if (template.isCustomJnlpContainerEnabled() && template.getJnlpContainer() != null) {
```

Comment:
hm.  So is there any way for me to avoid having a jnlp (custom or otherwise) added to my pod?  is setting the container name to jnlp the only way?